server/paths/refer.ts

```diff
@@ -49,6 +49,7 @@ export default {
     editProgramme: programmeHistoryProgrammePath,
     index: programmeHistoryPath,
     new: newProgrammeHistoryPath,
+    review: programmeHistoryPath,
```

Comment:
I wonder if `markReviewed` would make it clearer what this is doing? Or `updateHasReviewedProgrammeHistory` or `updateReviewedStatus` to keep it more generic (technically it could be used to mark it as not reviewed)

Reply:
Gone with `updateReviewedStatus`